cfpb/cfgov-refresh
# Ask tag page layout

Update Spanish tag page template

## Changes
- Update Spanish Ask page template

## Testing
- Check out branch
- Set up Ask if necessary
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Navigate to [Spanish tag page](http://localhost:8000/es/obtener-respuestas/buscar-por-etiqueta/hipotecas/?page=1) and verify contents/layout

## Screenshots


![tag-page](https://cloud.githubusercontent.com/assets/778171/26315646/a5d7047c-3ec6-11e7-8b34-aabeef923afc.png)


## Checklist
